pylintrc

```diff
@@ -9,7 +9,7 @@
 
 # Add files or directories to the blacklist. They should be base names, not
 # paths.
-ignore=CVS
+ignore=CVS,debug.py
```

Comment:
why did you remove this one?

Reply:
I didn't remove it, I added debug.py to the pylint ignore list

Reply:
No idea but it does not seem necessary at all.

Reply:
switched to single `# pylint: disable=import-error` on the import line and it seems to work fine.